shell/platform/common/cpp/BUILD.gn

```diff
@@ -46,7 +46,7 @@ source_set("common_cpp") {
   deps = [
     ":common_cpp_library_headers",
     "$flutter_root/shell/platform/common/cpp/client_wrapper:client_wrapper",
-    "$flutter_root/shell/platform/embedder:embedder",
+    "$flutter_root/shell/platform/embedder:embedder_with_symbol_prefix",
```

Comment:
You'd still need to keep the original version around since that is the one the bots use to distribute to gcloud. You'll probably want to disambiguate the dylibs on disk too.

Reply:
It's still present; that's flutter_engine_library (https://github.com/flutter/engine/blob/master/shell/platform/embedder/BUILD.gn#L98) which is unchanged.

Reply:
Ah, got it. Thanks for clarifying. Sounds great.